lib/e3db/client.rb

```diff
@@ -449,18 +537,26 @@ def each
           json = @client.instance_eval { query1(q) }
           results = json[:results]
           results.each do |r|
-            record = Record.new(meta: r[:meta], data: r[:record_data] || Hash.new)
-            if q.include_data && !@raw
-              access_key = r[:access_key]
-              if access_key
-                record = @client.instance_eval {
-                  ak = decrypt_eak(access_key)
-                  decrypt_record_with_key(record, ak)
-                }
+            if q.include_data
+              if !@raw
```

Comment:
Thought we were getting rid of the `raw` flag in queries?

Reply:
Will happen in a follow up PR (see https://github.com/tozny/e3db-ruby/tree/jgbailey/safe-delete  branch).

Reply:
:+1: